packages/controllers/src/controllers/ConnectionController.ts

```diff
@@ -379,7 +380,8 @@ const controller = {
       event: 'CONNECT_SUCCESS',
       properties: {
         method: wcLinking ? 'mobile' : 'qrcode',
-        name: RouterController.state.data?.wallet?.name || 'Unknown'
+        name: RouterController.state.data?.wallet?.name || 'Unknown',
+        chainId: ChainController.getActiveCaipNetwork()?.caipNetworkId as CaipNetworkId
```

Comment:
I'd prefer to avoid type casting, instead make the `chainId` accepts `undefined`